### ISSUE TITLE
Rename cmdline tools with consistent (and unique) prefix

### ISSUE THREAD

**Issue by yarikoptic** [NDAR/nda-tools#7]
I would strongly recommend to rename `downloadcmd` into something not that generic.  With that in mind I would recommend to rename them all to have `nda-` prefix, e.g.

- `nda-download` (instead of generic `downloadcmd`)
- `nda-upload` (instead of cryptic `vtcmd`)

**Comment by ushnaahmad** [comment]
Agreed, it would make things clearer, and all future command line tools added to the package could be prefixed with "nda-". I'm creating a ticket for us to change this in a future release.
